garage/sampler/stateful_pool.py

```diff
@@ -34,10 +33,8 @@ def initialize(self, n_parallel):
             self.manager = mp.Manager()
             self.queue = mp.Queue()
             self.worker_queue = mp.Queue()
-            self.pool = MemmappingPool(
-                self.n_parallel,
-                temp_folder="/tmp",
-            )
+            self.pool = mp.Pool(self.n_parallel)
```

Comment:
joblib seemed like an essential part of the original design. is there a reason you are removing this? are you sure it has no performance implications?

Reply:
I will revert it to use the original joblib pool. Tensorflow has a caveat of not being fork-safe. (See [here](https://github.com/rlworkgroup/garage/blob/7013b23a86d4b1eba159289c7e602aae9dc050cd/examples/tf/trpo_cartpole_batch_sampler.py#L22)) I was playing around to see if using `multiprocessing.Pool` would bring me any luck.